### ISSUE TITLE
[jira] consider adding dropdown for tempo account field

### ISSUE THREAD

**Issue by macqueen** [getsentry/sentry-plugins#88]
the custom field type is `com.tempoplugin.tempo-accounts:accounts.customfield`

**Comment by macqueen** [comment]
see https://forum.sentry.io/t/jira-problem-creating-issues-due-to-required-custom-field-not-being-passed-properly/462 for background

**Comment by benmcginnis** [comment]
@macqueen if I send a PR would you guys look at it?

**Comment by macqueen** [comment]
@benmcginnis yeah, definitely! just assign it to me and i'll make sure we take a look. thanks!
